cronjobs.yml

```diff
@@ -4,6 +4,9 @@ globalEnvSecrets:
   EVG_API_KEY: selected-tests-secrets
   SELECTED_TESTS_MONGO_URI: selected-tests-secrets
 cronJobs:
-  - name: process-work-items
+  - name: process-test-mappings
     schedule: "0 1 * * *"
     command: ["/bin/sh", "cronjobs/process_test_mapping_work_items.sh"]
+  - name: process-task-mappings
+    schedule: "0 13 * * *"
```

Comment:
[nit]  I would still probably have this happen over night, since there will probably be less load on all the systems.

So maybe,
```
0 2 * * * 
```